src/client/components/Sidebar/_index.scss

```diff
@@ -0,0 +1,21 @@
+.row {
```

Comment:
I'm a little worried that these styles will inadvertently affect other instances of navbars/rows. Could we nest these in a class called Sidebar?

Reply:
yikes, great point. I somehow assumed it was all magically namespaced within each component :P fixed!

only somewhat relatedly, I just discovered SASS has deprecated @import: https://sass-lang.com/blog/the-module-system-is-launched